task/runner.go

```diff
@@ -545,6 +546,97 @@ func (r *runner) Shutdown(ctx context.Context) error {
 	return r.amqp.Shutdown(ctx)
 }
 
+func (r *runner) UnpinFromIpfs(ctx context.Context, cid string, filter string) error {
```

Comment:
Can we move these new funcs to a separate file/struct? I think this runner is already doing a lot so we can keep business logic separate